code/modules/core_implant/cruciform/rituals/crusader.dm

```diff
@@ -54,7 +54,7 @@
 	phrase = "Per fidem enim ambulamus et non per speciem."
 	desc = "Knocks over everybody without cruciform in the view range. Psy-wave is too powerful, speaker can be knocked too."
 	cooldown = TRUE
-	cooldown_time = 2 MINUTES
+	cooldown_time = 10 SECONDS
```

Comment:
A target-specific flashbang seems a little strong to be able to fire it off every 10 seconds

Reply:
It's not really every ten seconds though, Cruciforms don't have cooldowns on things generally because stuff is gated behind cruciform power.

Reply:
I suddenly realise what you mean and I am a dumbass.